CHANGES.rst

```diff
@@ -4,7 +4,10 @@ astropy-helpers Changelog
 1.1.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added sphinx configuration value ``automodsumm_inherited_members``.
```

Comment:
This should be in the 1.2 section, as it's adding a new feature.  (although see my comment in astropy/astropy-helpers#216 about the idea of doing a 1.2 release of the helpers prior to the 1.2 release of astropy)